vpr/src/device/rr_graph_builder.h

```diff
@@ -234,7 +261,10 @@ class RRGraphBuilder {
     t_rr_graph_storage& node_storage_;
     /* Fast look-up for rr nodes */
     RRSpatialLookup node_lookup_;
-
+    /* Segment info for rr nodes */
+    const vtr::vector<RRSegmentId, t_segment_inf>& rr_segments_;
```

Comment:
@tangxifan 
Is rr_segments correctly added here ?

Reply:
It should be
```
vtr::vector<RRSegmentId, t_segment_inf> rr_segments_;
```

We use constant in ``RRGraphView`` but in ``RRGraphBuilder`` it should not (as it owns the storage)